apps/web/src/pages/SearchRequests/SearchPage/components/SearchResultCard.tsx

```diff
@@ -83,35 +109,26 @@ const SearchResultCard = ({ candidateCount, pool }: SearchResultCardProps) => {
       <p
         data-h2-display="base(flex)"
         data-h2-flex-wrap="base(wrap)"
-        data-h2-gap="base(x.125)"
+        data-h2-gap="base(0, x.5)"
+        data-h2-font-size="base(caption)"
       >
         {pool?.essentialSkills && pool?.essentialSkills.length > 0
-          ? pool.essentialSkills.map((skill) => (
-              <span key={skill.id}>
-                <Pill key={skill.id} color="secondary" mode="outline">
+          ? pool.essentialSkills.map((skill, index) => (
+              <>
```

Comment:
Looks like you might need a key.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/428fc601-72bc-483a-a6bb-f0a3663a450a)

Reply:
Oops, added with [Add key to mapped fragment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9139/commits/d38fa482dc0bae4228052a8698b67660138d9cdc).